test/strategy/auth0_test.exs

```diff
@@ -58,54 +63,66 @@ defmodule Ueberauth.Strategy.Auth0Test do
   end
 
   # Tests:
-  test "simple request phase", fixtures do
+  test "simple request phase" do
     conn =
       :get
       |> conn("/auth/auth0")
       |> SpecRouter.call(@router)
 
-    assert conn.resp_body == fixtures.response_basic
+    assert conn.resp_body =~ ~s|<html><body>You are being <a href=|
+    assert conn.resp_body =~ ~s|>redirected</a>.</body></html>|
+    assert conn.resp_body =~ ~s|href="https://example-app.auth0.com/authorize?|
+    assert conn.resp_body =~ ~s|client_id=clientidsomethingrandom|
+    assert conn.resp_body =~ ~s|redirect_uri=http%3A%2F%2Fwww.example.com%2Fauth%2Fauth0%2Fcallback|
+    assert conn.resp_body =~ ~s|response_type=code|
+    assert conn.resp_body =~ ~s|scope=openid+profile+email|
+    assert conn.resp_body =~ ~s|state=#{conn.private[:ueberauth_state_param]}|
   end
 
-  test "advanced request phase", fixtures do
+  test "advanced request phase" do
     conn =
       :get
       |> conn(
         "/auth/auth0?scope=profile%20address%20phone&audience=https%3A%2F%2Fexample-app.auth0.com%2Fmfa%2F" <>
-          "&state=obscure_custom_value&connection=facebook&unknown_param=should_be_ignored" <>
+          "&connection=facebook&unknown_param=should_be_ignored" <>
           "&prompt=login&screen_hint=signup&login_hint=user%40example.com"
       )
       |> SpecRouter.call(@router)
 
-    assert conn.resp_body == fixtures.response_advanced
```

Comment:
response_advanced looks redundant now right?

Reply:
Yup, and response_basic! I'll remove both of those and the corresponding files to clean it up.